microcosm_sqlite/factories.py

```diff
@@ -3,37 +3,54 @@
 
 """
 from distutils.util import strtobool
+from pkg_resources import iter_entry_points
 
 from microcosm.api import defaults
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
 
 @defaults(
-    path=":memory:",
-    # echo all SQL
     echo="False",
+    path=":memory:",
+    paths=dict(),
 )
-def configure_sqlite_engine(graph):
+class SQLiteBindFactory:
```

Comment:
Rather than having a factory that returns a single `engine` for the `graph.sqlite` binding, we have a factory that knows how to generate (and cache) the `engine` (and `sessionmaker`) by data set name.